composables/useValidateEmail.ts

```diff
@@ -14,5 +14,11 @@ export const useValidateEmail = () => {
     return emailRegex.test(email);
   };
 
-  return { validateEmail, emailError, emailTouched };
+  const handleEmailInput = () => {
```

Comment:
The idea is that if the user enters invalid value and on blur we get the error, but if the user comes back to the field and starts typing again - the error is cleared - cuz there is a chance the value will be correct :), validation is again on blur. 
This might not be the best solution - feel free to propose me sth (but to be fixed in other PR, cuz I want to merge this one :))